### DISCUSSION TITLE
Missing Actions?

### DISCUSSION THREAD

**Discussion by YeapGuy** [File-New-Project/EarTrumpet#1265] [Q&A] [answered]
Hi, just upgraded to latest, but I can't find the Actions I see mentioned everywhere in other issues, discussions..? I only have "General" in Settings, and under there 3 tabs - Shortcuts, Legacy and About.

Where's the Actions config? I want to create shortcuts for switching of audio sources, which I guess is now possible.

**Comment by riverar** [comment, marked as answer]
> Hi, just upgraded to latest, but I can't find the Actions I see mentioned everywhere in other issues, discussions..? I only have "General" in Settings, and under there 3 tabs - Shortcuts, Legacy and About.
> 
> Where's the Actions config? I want to create shortcuts for switching of audio sources, which I guess is now possible.

Hey @YeapGuy that's in our dev builds (link on our https://eartrumpet.app page)! Install one of those and give it a whirl.

**Reply by YeapGuy** [reply]
But...
![image](https://user-images.githubusercontent.com/43749899/200912550-ad3d7797-fc6f-44fa-9fce-68f5138a5a93.png)
Isn't 2.2.1.0 up to par with the dev branch?

**Reply by riverar** [reply]
EarTrumpet doesn't spin up addons (or Actions) if the package doesn't have developer identity (i.e. is a dev build) https://github.com/File-New-Project/EarTrumpet/blob/master/EarTrumpet/App.xaml.cs#L93

**Reply by YeapGuy** [reply]
Ah, I see! Thanks for the explanation.
